features/file-system-access.yml

```diff
@@ -4,8 +4,6 @@ spec: https://wicg.github.io/file-system-access/
 caniuse: native-filesystem-api
 group: file-system
 compat_features:
-  # api.DataTransferItem.getAsFileSystemHandle is likely to be part of a drag and drop feature, but it can't be in both features at once. See https://github.com/web-platform-dx/web-features/issues/1173 for details
```

Comment:
I agree it makes more sense in Drag and drop, but should we move this comment there?

Reply:
Good call, I moved it over and reworded.